src/distributions/other.rs

```diff
@@ -32,19 +32,21 @@ use std::mem::{self, MaybeUninit};
 /// # Example
 ///
 /// ```
-/// use std::iter;
 /// use rand::{Rng, thread_rng};
 /// use rand::distributions::Alphanumeric;
 ///
 /// let mut rng = thread_rng();
-/// let chars: String = iter::repeat(())
-///         .map(|()| rng.sample(Alphanumeric))
-///         .map(char::from)
-///         .take(7)
-///         .collect();
+/// let chars: String = (0..7).map(|_| rng.sample(Alphanumeric) as char).collect();
 /// println!("Random chars: {}", chars);
 /// ```
 ///
+/// Alternatively, one can use the [`DistString`] trait:
```

Comment:
Maybe mention that this is more efficient?

Reply:
Updated